<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_create_txt_and_json">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Create TXT &amp;&amp; JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Clear List</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_film_eng_name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>280</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_per">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Create Per Name &amp;&amp; Date from Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_film_per_name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>280</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Per Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Eng Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_film_date">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_copy_folder_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);</string>
    </property>
    <property name="text">
     <string>Copy Folder Name</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_doostiha_date">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Create Doostiha Date TXT &amp;&amp; JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_txt_from_json">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Create TXT From JSON</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_hidden">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_filter_per_names">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Filter Empty Per Names</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_filter_eng_names">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Filter Empty Eng Names</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_filter_dates">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Filter Empty Dates</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>lineEdit_film_eng_name</zorder>
   <zorder>pushButton_create_txt_and_json</zorder>
   <zorder>pushButton_clear</zorder>
   <zorder>label_name</zorder>
   <zorder>pushButton_filter_per_names</zorder>
   <zorder>pushButton_filter_dates</zorder>
   <zorder>pushButton_create_per</zorder>
   <zorder>pushButton_filter_eng_names</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_film_date</zorder>
   <zorder>label_3</zorder>
   <zorder>checkBox_copy_folder_name</zorder>
   <zorder>lineEdit_film_per_name</zorder>
   <zorder>pushButton_create_doostiha_date</zorder>
   <zorder>pushButton_create_txt_from_json</zorder>
   <zorder>listWidget_hidden</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_film_per_name</tabstop>
  <tabstop>lineEdit_film_eng_name</tabstop>
  <tabstop>lineEdit_film_date</tabstop>
  <tabstop>pushButton_filter_per_names</tabstop>
  <tabstop>pushButton_filter_dates</tabstop>
  <tabstop>pushButton_create_per</tabstop>
  <tabstop>pushButton_filter_eng_names</tabstop>
  <tabstop>pushButton_create_txt_and_json</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>checkBox_copy_folder_name</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
